lib/seahorn/BvOpSem2.cc

```diff
@@ -885,6 +886,26 @@ class OpSemMemManager {
     return res;
   }
 
+  /// \brief Executes symbolic memcpy from physical memory with concrete length
+  Expr MemCpy(PtrTy dPtr, char* sPtr, unsigned len, uint32_t align) {
+    Expr res = m_ctx.read(m_ctx.getMemReadRegister());
+    unsigned sem_word_sz = wordSzInBytes();
+    for (int i = 0; i < len; i += sem_word_sz) {
+      Expr offset = bv::bvnum(i, ptrSzInBits(), m_efac);
+      Expr dIdx = i == 0 ? dPtr : mk<BADD>(dPtr, offset);
+      unsigned val32 = 0;
+      for (int byte = sem_word_sz - 1; byte >= 0; byte--) {
```

Comment:
I think you can do this with `memcpy` without unaligned memory accesses. Minor, but will look cleaner